widget/entry.go

```diff
@@ -102,6 +105,7 @@ type Entry struct {
 
 // NewEntry creates a new single line entry widget.
 func NewEntry() *Entry {
+	//	e := &Entry{Wrapping: fyne.TextTruncate, PreeditText: "(日本語)"}
```

Comment:
Please don't leave dead code in comments

Reply:
@andydotxyz OK, removed